src/ServiceControlInstaller.Engine/Instances/ServiceControlAuditNewInstance.cs

```diff
@@ -0,0 +1,96 @@
+﻿namespace ServiceControlInstaller.Engine.Instances
+{
+    using System;
+    using System.IO;
+    using System.Linq;
+    using System.Reflection;
+    using System.Xml;
+    using System.Xml.Serialization;
+    using Configuration.ServiceControl;
+    using FileSystem;
+    using Queues;
+    using Services;
+    using Validation;
+
+    public class ServiceControlAuditNewInstance : ServiceControlInstallableBase, IServiceControlAuditInstance
+    {
+        public static ServiceControlAuditNewInstance CreateWithDefaultPersistence()
+        {
+            return CreateWithDefaultPersistence(Assembly.GetExecutingAssembly().Location);
+        }
+
+        public static ServiceControlAuditNewInstance CreateWithDefaultPersistence(string deploymentCachePath)
+        {
+            var zipInfo = ServiceControlAuditZipInfo.Find(deploymentCachePath);
+            var persistenceManifest = ServiceControlAuditPersisters.LoadAllManifests(zipInfo.FilePath)
```

Comment:
We seem to frequently load these manifests. I wonder if we should introduce an `AuditInstanceFactory` or something from the Deploy Cache Path that can load this stuff once and cache it.